src/Analyzers/CSharp/Tests/UseExpressionBody/UseExpressionBodyForPropertiesAnalyzerTests.cs

```diff
@@ -533,5 +533,29 @@ int Bar
 }";
             await TestWithUseExpressionBody(code, fixedCode, LanguageVersion.CSharp6);
         }
+
+        [WorkItem(50181, "https://github.com/dotnet/roslyn/issues/50181")]
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsUseExpressionBody)]
+        public async Task TestUseExpressionBodyPreserveComments()
+        {
+            var code = @"
+public class C
+{
+    {|IDE0025:public long Length                   //N
+    {
+        // N = N1 + N2
+        get { return 1 + 2; }
+    }|}
+}";
+            var fixedCode = @"
+public class C
+{
+    public long Length                   //N
+
```

Comment:
this doesn't look like its quite correct - we don't want this additional empty line here - we might be appending too much trivia